lib/process.js

```diff
@@ -181,13 +181,18 @@ function processOperations(api, parentPath, topLevel, options) {
 function processTransactions(api, parentOp, parentPath, topLevel, options) {
     var transactions = []
 
+    // Replace newlines with a space
+    var stripNewlines = function(str) {
```

Comment:
Perfect, moved the function to the bottom (and renamed it to `replaceNewlines` to be more indicative of the actual operation).